src/raft_server.c

```diff
@@ -6061,6 +6061,7 @@ raft_server_backend_close(struct raft_instance *ri)
     if (!ri || !raft_instance_is_shutdown(ri))
         return -EINVAL;
 
+    SIMPLE_LOG_MSG(LL_WARN, "RAFT_SERVER_BACKEND_CLOSE: Calling backend shutdown (normal shutdown path)");
```

Comment:
`SIMPLE_LOG_MSG()` will print the function name in the msg so there's no need to add it in the msg!